lib/managers/legal-hold-policies.js

```diff
@@ -64,6 +64,7 @@ LegalHoldPolicies.prototype.assignmentTypes = Object.freeze({
  * @param {string} [options.description] - Description of the legal hold policy
  * @param {string} [options.filter_started_at] - Date filter, any Custodian assignments will apply only to file versions created or uploaded inside of the date range
  * @param {string} [options.filter_ended_at] - Date filter, any Custodian assignments will apply only to file versions created or uploaded inside of the date range
+ * @param {boolean} [options.is_ongoing] - After initialization, Assignments under this Policy will continue applying to files based on events, indefinitely
```

Comment:
This isn't related to the rest of the change; please don't mix together separate changes if possible.